vdc/k8s-config.yaml

```diff
@@ -271,3 +271,55 @@ spec:
   resources:
     requests:
       storage: 10Mi
+---
+apiVersion: v1
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: test
+  name: delete-test-pvc
+rules:
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["delete"]
+---
+apiVersion: v1
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: deploy-svc-delete-test-pvc
+  namespace: test
+subjects:
+- kind: ServiceAccount
+  name: deploy-svc
+  namespace: batch-pods
+roleRef:
+  kind: Role
+  name: delete-test-pvc
+  apiGroup: "rbac.authorization.k8s.io"
+---
+apiVersion: v1
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: batch-pods
+  name: delete-batch-pods-pvc
+rules:
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["delete"]
+---
+apiVersion: v1
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: deploy-svc-delete-batch-pods-pvc
+  namespace: batch-pods
```

Comment:
Can this be omitted, since it matches the namespace defined in the delete-batch-pods-pvc Role (genuinely not sure)?

Reply:
It seems that the namespace of a RoleBinding overrides the namespace of a Role? If so, when we mean to use the namespace defined on the Role, can we omit the namespace?